src/unity/python/turicreate/cython/cy_flexible_type.pyx

```diff
@@ -534,10 +593,6 @@ cdef __fill_numpy_info():
     for c in <str>(__typecodes.get("Character", "") + "SU"):
         _dtype_to_flex_enum_lookup[<int>(ord(c))] = STRING
 
-cdef type np_ndarray
```

Comment:
The reason this was left here is for speed.  import numpy as np goes through python's module and dict lookup system, this is a local C variable.  Thus if we refer to np_ndarray, it avoids the dictionary lookup performed to get the type value.